### ISSUE TITLE
plot(section) can error if some pressures are NA

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1006]
I ran across this with some MEOP data. When the section plot code examines profiles to see if they are all "equal" in pressure, it can error when comparing values that are `NA`. The error is as below.
```R
Error in if ("points" != ztype && np1 != length(thisPressure) || any(p1 !=  : 
  missing value where TRUE/FALSE needed
```

**Comment by dankelley** [comment]
I've grabbed a sample file that demonstrates it, and it and test code are now in the [oce-issues repo](https://github.com/dankelley/oce-issues/blob/master/1006).

**Comment by dankelley** [comment]
I've fixed the original problem but there is also a problem with salinities being NA, so I'll fix that also. (Possibly I'll report it and close the report a moment later, for the record).

**Comment by dankelley** [comment]
Fixed in `develop` commit 55b523eb63fb97767091cde1686a45d619d0d531
